whois-api/src/main/java/net/ripe/db/whois/api/rest/ClientCertificateExtractor.java

```diff
@@ -23,6 +24,9 @@
     private final static String HEADER_SSL_CLIENT_CERT = "SSL_CLIENT_CERT";
     private final static String HEADER_SSL_CLIENT_VERIFY = "SSL_CLIENT_VERIFY";
 
+    private static final String X509_HEADER = "-----BEGIN CERTIFICATE-----";
```

Comment:
Can you use the X509CertificateWrapper class to extract the certificate?

Reply:
Done.